.github/workflows/goreleaser.yaml

```diff
@@ -0,0 +1,60 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  goreleaser:
+    environment: arlon
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.17
```

Comment:
Can we use go 1.17 for 0.3 tags and go 1.18 / 1.19 for other tags including main, 0.9 once go 1.18 is verified and approved?

Reply:
I'll tinker in the hack repo and let you know

Reply:
https://github.com/arlonproj/arlon/pull/180/files
This is what I've come up with. Feel free to drop the PR if that's not what we want